sites/fast-color-explorer/app/app.ts

```diff
@@ -85,6 +85,17 @@ const colorBlockTemplate = html<App>`
 
 const template = html<App>`
     <div class="container fill">
+        <div class="row banner">
+            <p>
+                <a
+                    href="https://aka.ms/fluentwebcomponents/?path=/docs/design-system-color-explorer--page"
+                >
+                    Fluent UI Web Components color explorer
+                </a>
+                has moved to the Fluent UI Storybook site. This site now represents the
+                latest version of @microsoft/fast-components.
```

Comment:
Should this note "deprecated" instead of "latest"? Or perhaps we say, "This site now represents the final version of `@microsoft/fast-components`"?

I assume the color explorer will ultimately end up representing Adaptive UI once stable?